content/projekte/archiv/_index.md

```diff
@@ -8,3 +8,5 @@ title: Archiv
 {{< headline-brackets-h1 color="red"  >}}
 Projektarchiv
```

Comment:
```suggestion
Alle Projekte
```

Reply:
Umbenennen deshalb weil: wir nutzen die Seite ja nicht mehr als Archiv, wo alte Projekte sind, sondern als generelle Übersichtsseite für alle Projekte. 

Nice to have, optional: Nur falls es ganz leicht möglich ist nicht viel Arbeit nach sich zieht, wäre es auch schön wenn sich der Link von projekte/**archiv** ändert auf projekte/**alle**.